frontend/sharedServices/BUILD

```diff
@@ -0,0 +1,21 @@
+load("//tools/lint:web.bzl", "web_lint")
+load("@com_github_airyhq_bazel_tools//lint:buildifier.bzl", "check_pkg")
+load("@com_github_airyhq_bazel_tools//web:typescript.bzl", "ts_web_library")
+
+package(default_visibility = ["//visibility:public"])
```

Comment:
I get the intention of this module, but I think the name is too generic and not quite accurate (It hosts a component and not a service). Only thing I can think of for now would be `frontend/components`

Reply:
`frontend/lib/avatar` would be my suggestion (given the previous point I made about it)